scripts/ccf/acl/up.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+REPO_ROOT="$(realpath "$(dirname "$(realpath "${BASH_SOURCE[0]}")")/../../..")"
+
+acl-up() {
```

Comment:
acl-create

Reply:
All other ccf services in this repo use `up.sh` and `down.sh` scripts which is what the test framework uses to find the code to bring networks up and down.

The choice of up/down comes from the fact the scripts also handle cases where the desired state already exists and therefore no creation or deletion is required.